atlp-rwanda/atlp-apprenticeship-bn
# setting up docker for backend

#### What does this PR do?
setup docker compose for our backend project

#### Description of Task to be completed?
As a developer, I want to setup  docker so that new images can be created for deployment.

```
How Can I test it


git clone https://github.com/atlp-rwanda/atlp-apprenticeship-bn.git
cd atlp-apprenticeship-bn
git checkoutch-settingUp-Docker
npm install

Intall Docker desktop in your PC and make sure all the settings is done and you can be able to open it.

Go back to your VS code terminal run the following commands

1. docker build -t <<the name of the image>> . 
2. docker image ls  /* for listing all image that you have */
3. docker run -p 4000:4000 -d --name   /*the name of the container*/   /*name of the image*/
4. docker ps  /*This is to see the container opened at this moment*/
5.  Go to your Browser and run localhost:4000
You should be able to run server




```

#### What are the relevant pivotal tracker/Trello stories?

Trello [Link]() (#-https://trello.com/c/axA4L3dX)

#### Screenshots (if appropriate)  )

||
![ApprenticeshipTwo](https://github.com/atlp-rwanda/atlp-apprenticeship-bn/assets/86248137/27908881-da2c-4dc7-99df-3f9d19161604)
||
![Docker](https://github.com/atlp-rwanda/atlp-apprenticeship-bn/assets/86248137/93b6958d-b591-4425-ae29-88bd78a8a5f8)
||
![DockerTwo](https://github.com/atlp-rwanda/atlp-apprenticeship-bn/assets/86248137/aef2962f-0ef6-4a52-a72c-3feb6443141d)
